docker-compose.yml

```diff
@@ -2,25 +2,21 @@ services:
 
   web:
     build:
-      context: ./ccm_web/
-      dockerfile: ./Dockerfile
+      context: .
+      dockerfile: Dockerfile
       args:
```

Comment:
I don't think the args are needed here anymore.

Reply:
Thanks, Removed